Kickstarter-iOS/Features/PledgedProjectsOverview/PPOView.swift

```diff
@@ -28,17 +28,20 @@ struct PPOView: View {
     }
   }
 
+  @ViewBuilder func listViewHeader(numberOfValues: Int) -> some View {
+    Text(Strings.Alerts_count(count: numberOfValues.formatted()))
+      .font(Font(UIFont.ksr_title2()))
+      .background(Color(UIColor.ksr_white))
+      .foregroundStyle(Color(UIColor.ksr_black))
+      .padding(.top)
```

Comment:
would it be helpful to include padding values in `PPOCardStyles` as well as colors and fonts?

Reply:
Done!